index.css

```diff
@@ -0,0 +1,67 @@
+@import url('/quizQuestions/questionStyle.css');
+@import url('/quizQuestions/result.css');
+
+:root{
+    --spacing-one:1rem;
+    --spacing-two:2rem;
+    --spacing-three:4rem;
```

Comment:
`--spacing-three: 4rem;`
wouldn't it be hard to remember the variables? having `three` in name and linking to `4 rem`.
why not use it like this:
`--spacing-4: 4rem;`
`--spacing-6: 6rem;`
`--spacing-8: 8rem;`
...

Reply:
it's in multiple of two and enumerable from one.

Reply:
Talked with the dev and got the rationale behind the naming convention. 
Good to merge

Reply:
I have changed the naming as the changes suggested were good. Thankyou 😀